prow-jobs/pingcap-tiproxy-latest-postsubmits.yaml

```diff
@@ -0,0 +1,47 @@
+# struct ref: https://pkg.go.dev/k8s.io/test-infra/prow/config#Presubmit
+presubmits:
+  pingcap/TiProxy:
+    - name: pingcap/tiproxy/merged_integration_prisma_test
+      agent: jenkins
+      decorate: false # need add this.
+      skip_if_only_changed: "(\\.md|Dockerfile|OWNERS|OWNERS_ALIASES)$"
+      always_run: false
+      context: merged-integration-prisma-test # need change this.
+      skip_report: false
+      trigger: "(?m)^/test (?:.*? )?merged-integration-prisma-test(?: .*?)?$"
+      rerun_command: "/test merged-integration-prisma-test"
```

Comment:
These two configurations(`trigger` and `rerun_command`) are not required in the postsubmit job, please remove them.